components/internal/FederationNotice.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 	<section v-if="unsupported.length > 0" class="vue-component message-block federation federation-backends">
 		<strong class="header">Federation</strong>
-		<p>This {{ entity }} is only available through the following services in the federation:</p>
```

Comment:
Here's a clarification on why the "only" was there and should be kept:
https://github.com/Open-EO/openeo-vue-components/issues/101#issuecomment-2793393936